config/locales/thematic_area/marine/en.yml

```diff
@@ -15,3 +15,23 @@ en:
         chart_text: "%{country} and its %{overseas} overseas countries and territories have a combined area of %{km}km<sup>2</sup>"
         chart_stat_text: of their national waters are protected
         chart_stat_text_overseas: of their overseas territories waters are protected
+
+      social:
+        title: Explore the World's Marine Protected Areas
```

Comment:
It should be doing this, it depends on whether they have been set or not~

The bits loaded from locales are default settings

Reply:
Ah, this is because marine_controller.rb has a bespoke override, which stems from what was there before my work on this branch began, so it would perhaps be redundant to have this custom override now: https://github.com/unepwcmc/ProtectedPlanet/blob/ticket-54/app/controllers/marine_controller.rb#L27

This can probably be removed if the content is now in the CMS directly.

Reply:
If this is redundant, I can remove it and the locale references too? Should theme_image be set for the twitter:card as well? I am sure that Twitter will use og:image by default if not provided though.